R/students.R

```diff
@@ -1752,7 +1749,7 @@ s_35 <- function(input_df=usheUtils::fake_student_df, with_intermediates=FALSE)
 
   output_df <- input_df %>%
     # Calculate intermediate fields
-    mutate( s_banner_id = student_id) %>%
+    mutate( s_banner_id = paste0('D', student_id )) %>%
```

Comment:
**Just Ideas:**

This would be a useful piece of code to have in {utHelpR}.

Something like: make_d_number( student_id )

This function could also convert integers to strings and pad that string with '0'.

Reply:
Good suggestion!